src/encoded/schemas/functional_characterization_experiment.json

```diff
@@ -236,36 +236,26 @@
                 }
             ]
         },
-        "gRNA_tiling_loci": {
-            "title": "gRNA tiling loci",
-            "description": "The loci targeted by gRNAs.",
-            "type": "array",
-            "uniqueItems": true,
-            "minItems": 2,
-            "items": {
-                "title": "Locus",
-                "description": "Gene locus targeted by gRNAs.",
-                "type": "string",
-                "linkTo": "Gene"
-            }
-        },
         "examined_loci": {
             "title": "Examined loci",
             "description": "The loci whose expression or product is under investigation for the functional characterization experiment.",
             "type": "array",
             "uniqueItems": true,
-            "minItems": 2,
+            "minItems": 1,
             "items": {
                 "title": "Locus",
                 "description": "The gene whose expression or product is under investigation for the functional characterization experiment.",
                 "type": "object",
                 "additionalProperties": false,
                 "anyOf": [
```

Comment:
probably should be oneOf and not anyOf

Reply:
Addressed; see JIRA ticket for detailed comment. Needed to add another restriction to the gene-only case